### ISSUE TITLE
Making server start window more minimal in intellij plugin.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#191]
```
Hello,

I can see your icons too big and it is unnecessary to make a new UI window for 
that in intellij. You can just add start/stop button to bottom right of status 
bar and browser icons you captured. So, we dont need to see extra window or big 
icons that are useless.

Thank you:)
```

Original issue reported on code.google.com by `fatihkad...@gmail.com` on 23 Jan 2011 at 1:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 23 Jan 2011 at 1:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 11 May 2011 at 7:43
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Yes, I agree with you, this tool window should be redesign.
```

Original comment by `Sergey.S...@gmail.com` on 8 Jun 2011 at 1:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 117.65 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 27 Mar 2012 at 9:32
- Changed state: **Fixed**
